internal/config/config_defaults.go

```diff
@@ -0,0 +1,77 @@
+package config
+
+var ConfigDefaults Config
```

Comment:
Nit: No need to prefix it with `Config` because it's already in the `config` package. In my other comment, I suggested to hide it as you have `Defaults()` function.

```suggestion
var defaults Config
```